### PR TITLE
Cutadapt 1.8.1 goolf 1.7.20 python 2.7.9

### PR DESCRIPTION
**Pull request by npklein**
depends on #1703

### CODE REVIEW

File: easybuild/easyconfigs/c/cutadapt/cutadapt-1.8.1-goolf-1.7.20-Python-2.7.9.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
```
Comment (boegel) [inline easybuild/easyconfigs/c/cutadapt/cutadapt-1.8.1-goolf-1.7.20-Python-2.7.9.eb]: drop the empty line here?

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by verdurin** [comment]
Worth noting that #2569 has been merged. I would be willing to port it to goolf, if desired.

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/c/cutadapt/cutadapt-1.8.1-goolf-1.7.20-Python-2.7.9.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+pyshortver = '.'.join(pyver.split('.')[:2])
+
+versionsuffix = "-%s-%s" % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+]
+
+sanity_check_paths = {
+    'files': ['bin/cutadapt','lib/python%s/site-packages/cutadapt/_align.so' % pyshortver],
```
Comment (boegel) [inline easybuild/easyconfigs/c/cutadapt/cutadapt-1.8.1-goolf-1.7.20-Python-2.7.9.eb]: shouldn't be using hardcoded `.so`, but `.../_align.%s' % (pyshortver, SHLIB_EXT)` instead

**Comment by boegel** [comment]
@verdurin: a reworked version of this based on your PR that got merged makes sense

**Comment by boegel** [comment]
superseded by #6319, I don't think this is still relevant...

**Pull request closed without merging**